src/Core/Form/ChoiceProvider/TranslationTypeChoiceProvider.php

```diff
@@ -54,7 +54,7 @@ public function getChoices()
     {
         return [
             $this->translator->trans('Back office translations', [], 'Admin.International.Feature') => 'back',
-            $this->translator->trans('Themes translations', [], 'Admin.International.Feature') => 'themes',
+            $this->translator->trans('Front office Translations', [], 'Admin.International.Feature') => 'themes',
```

Comment:
Maybe use the lower caps t as in "Back office translations"
```suggestion
            $this->translator->trans('Front office translations', [], 'Admin.International.Feature') => 'themes',
```

Reply:
My mistake, I see that the wording already exists

Reply:
I checked, the key of translation in crowdin is with capital T
Maybe we can change it @Julievrz ?